.github/workflows/info.yml

```diff
@@ -0,0 +1,47 @@
+name: PR Lint keyboards
+
+on:
+  pull_request:
+    paths:
+    - 'keyboards/**'
+
+jobs:
+  info:
+    runs-on: ubuntu-latest
+
+    container: qmkfm/base_container
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Print info
+      run: |
+        git rev-parse --short HEAD
+        echo ${{ github.event.pull_request.base.sha }}
+        git diff --name-only ${{ github.event.pull_request.base.sha }}...
+
+    - name: Run qmk info
+      shell: 'bash {0}'
+      run: |
+        QMK_CHANGES=$(git diff --name-only ${{ github.event.pull_request.base.sha }}...)
+        QMK_KEYBOARDS=$(qmk list-keyboards)
+
+        exit_code=0
+        for KB in $QMK_KEYBOARDS; do
+          KEYBOARD_CHANGES=$(echo "$QMK_CHANGES" | grep -E '^(keyboards/'${KB}'/)')
+          if [[ -z "$KEYBOARD_CHANGES" ]]; then
+            # skip as no changes for this keyboard
+            continue
+          fi
+
+          KEYMAP_ONLY=$(echo "$KEYBOARD_CHANGES" | grep -cv /keymaps/)
```

Comment:
Should we include `default`, though?

Reply:
So from the discord chat, long term yes when we have more json keymaps. However right now it seems like it will be okay to ignore this case in the short term.